exec/vector/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -539,7 +553,12 @@ public void setValueLengthSafe(int index, int length) {
     }
 
 
-    public void setSafe(int index, int start, int end, DrillBuf buffer){
+<#if type.minor == "VarDecimal">
```

Comment:
I suppose condition should be negated or if and else should be swapped

Reply:
Is this what you're talking about?  It looks OK to me.
<#if type.minor == "VarDecimal">
    public void setSafe(int index, int start, int end, DrillBuf buffer, int scale)
<#else>
    public void setSafe(int index, int start, int end, DrillBuf buffer)
</#if> <#-- type.minor -->